ros_ws/src/car_control/src/dms_controller.cpp

```diff
@@ -11,7 +11,19 @@ DMSController::DMSController()
         this->m_node_handle.subscribe<std_msgs::Int64>(TOPIC_DMS, 1, &DMSController::dmsCallback, this);
 
     this->m_command_pulisher = this->m_node_handle.advertise<std_msgs::String>(TOPIC_COMMAND, 1);
-}
+    
+    m_node_handle.getParam(PARAMETER_DMS_CHECK_RATE, this->dms_check_rate);
```

Comment:
```suggestion
    ros::NodeHandle private_node_handle("~");
    private_node_handle.getParam(PARAMETER_DMS_CHECK_RATE, this->dms_check_rate);
```

Reply:
The nodehandler is a member variable. I tested it and it works fine.